src/ssl.c

```diff
@@ -45677,6 +45678,29 @@ int wolfSSL_SESSION_print(WOLFSSL_BIO *bp, const WOLFSSL_SESSION *x)
 #if defined(OPENSSL_ALL) || (defined(OPENSSL_EXTRA) && defined(HAVE_STUNNEL)) \
     || defined(WOLFSSL_MYSQL_COMPATIBLE) || defined(WOLFSSL_NGINX)
 
+/* TODO: Doesn't currently track SSL_VERIFY_CLIENT_ONCE */
+int wolfSSL_get_verify_mode(WOLFSSL* ssl) {
+    int mode = 0;
+    WOLFSSL_ENTER("wolfSSL_get_verify_mode");
+
+    if(!ssl)
+        return WOLFSSL_FATAL_ERROR;
```

Comment:
It doesn't look like OpenSSL is returning a -1 on failure. We should return WOLFSSL_FAILURE on an error. Although on a NULL input OpenSSL will SEGFAULT so I guess that this is undefined behaviour. Same issue in `wolfSSL_CTX_get_verify_mode`. OpenSSL implementation for reference:
```
int SSL_get_verify_mode(const SSL *s)
{
    return s->verify_mode;
}
```
Also, missing space after the `if`.

Reply:
Added space and changed to retrun `WOLFSSL_FAILURE`